src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.cpp

```diff
@@ -0,0 +1,658 @@
+                   outgoing_null_one_form.get(a) *
+                   outgoing_null_one_form.get(b) -
+               incoming_null_vector.get(c) * projection_Ab.get(d, a) *
+                   outgoing_null_one_form.get(b) -
+               incoming_null_vector.get(c) * projection_Ab.get(d, b) *
+                   outgoing_null_one_form.get(a) -
+               incoming_null_vector.get(d) * projection_Ab.get(c, a) *
+                   outgoing_null_one_form.get(b) -
+               incoming_null_vector.get(d) * projection_Ab.get(c, b) *
+                   outgoing_null_one_form.get(a) +
+               2. * projection_AB.get(c, d) * projection_ab.get(a, b)) *
+              char_projected_rhs_dt_v_minus.get(c, d);
+        }
+      }
+      for (size_t c = 0; c <= VolumeDim; ++c) {
+        bc_dt_v_minus->get(a, b) +=
+            0.5 * char_speeds[3] *
+            (constraint_char_zero_minus.get(c) -
+             mu * constraint_char_zero_plus.get(c)) *
+            (0.5 * outgoing_null_one_form.get(a) *
+                 outgoing_null_one_form.get(b) * incoming_null_vector.get(c) +
+             projection_ab.get(a, b) * outgoing_null_vector.get(c) -
+             projection_Ab.get(c, b) * outgoing_null_one_form.get(a) -
+             projection_Ab.get(c, a) * outgoing_null_one_form.get(b));
+      }
+    }
+  }
+}
+
+template <size_t VolumeDim, typename DataType>
+void add_physical_dof_terms_to_dt_v_minus(
+    const gsl::not_null<tnsr::aa<DataType, VolumeDim, Frame::Inertial>*>
+        bc_dt_v_minus,
+    const Scalar<DataType>& gamma2,
+    const tnsr::i<DataType, VolumeDim, Frame::Inertial>&
+        unit_interface_normal_one_form,
+    const tnsr::I<DataType, VolumeDim, Frame::Inertial>&
+        unit_interface_normal_vector,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>&
+        spacetime_unit_normal_vector,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>& projection_ab,
+    const tnsr::Ab<DataType, VolumeDim, Frame::Inertial>& projection_Ab,
+    const tnsr::AA<DataType, VolumeDim, Frame::Inertial>& projection_AB,
+    const tnsr::II<DataType, VolumeDim, Frame::Inertial>&
+        inverse_spatial_metric,
+    const tnsr::ii<DataType, VolumeDim, Frame::Inertial>& extrinsic_curvature,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>& spacetime_metric,
+    const tnsr::AA<DataType, VolumeDim, Frame::Inertial>&
+        inverse_spacetime_metric,
+    const tnsr::iaa<DataType, VolumeDim, Frame::Inertial>&
+        three_index_constraint,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>&
+        char_projected_rhs_dt_v_minus,
+    const tnsr::iaa<DataType, VolumeDim, Frame::Inertial>& phi,
+    const tnsr::ijaa<DataType, VolumeDim, Frame::Inertial>& d_phi,
+    const tnsr::iaa<DataType, VolumeDim, Frame::Inertial>& d_pi,
+    const std::array<DataType, 4>& char_speeds) noexcept {
+  // hard-coded value from SpEC Bbh input file Mu = MuPhys = 0
+  constexpr double mu_phys = 0.;
```

Comment:
could you also either remove `mu_phys` or hide all the math behind an `if-constexpr`?

Reply:
I think this still needs to be done

Reply:
hidden terms prop to `mu_phys` behind an `if constexpr`